gradle/scans-init.gradle.kts

```diff
@@ -0,0 +1,48 @@
+import com.gradle.enterprise.gradleplugin.GradleEnterprisePlugin
+import com.gradle.scan.plugin.BuildScanPlugin
+import java.io.ByteArrayOutputStream
+
+/**
+ * We want to publish scans only for top level build and when explicitly selected.
```

Comment:
why not enable it for all builds? or only ci builds using `publishAlwaysIf { }`

Reply:
> or only ci builds

That was actually my intention. I wanted to avoid publishing local builds. I agree this comment is misleading, I did not update it after some refactor I did